testlib/src/main/java/com/diffplug/spotless/ResourceHarness.java

```diff
@@ -42,6 +42,12 @@
 import com.diffplug.common.io.Resources;
 
 public class ResourceHarness {
+
+	/**
+	 * default test path.
+	 */
+	protected static String path = "src/main/java/test.java";
```

Comment:
```suggestion
	protected String path = "src/main/java/test.java";
```
this is kind of imposes a risk of collision, but it seems to work. Ideally would be `final` as well.

Reply:
```suggestion
	protected static final String DEFAULT_TEST_PATH = "src/main/java/test.java";
```
As per Java constant naming conventions.